include/tables/DatasetTable.h

```diff
@@ -134,21 +132,44 @@ class DatasetTable : public DBTable<DatasetRow> {
   DatasetRow getRow(NdbRecAttr* values[]) {
     DatasetRow row;
     row.mId = values[0]->int32_value();
-    row.mInodeId = values[1]->int64_value();
-    row.mInodeParentId = values[2]->int64_value();
-    row.mInodeName = get_string(values[3]);
-    row.mProjectId = values[4]->int32_value();
-    row.mDescription = get_string(values[5]);
-    row.mPublicDS = values[6]->int8_value() == 1;
+    row.mDatasetName = get_string(values[1]);
+    row.mProjectId = values[2]->int32_value();
+    row.mDescription = get_string(values[3]);
+    row.mPublicDS = values[4]->int8_value() == 1;
     return row;
   }
 
   DatasetRow get(Ndb* connection, int datasetId) {
     DatasetRow ds = doRead(connection, datasetId);
-    DatasetProjectSCache::getInstance().add(ds.mInodeId, ds.mProjectId, ds.mInodeName);
+    DatasetProjectSCache::getInstance().add(ds.mId, ds.mProjectId, ds.mDatasetName);
     return ds;
   }
 
+  DatasetRow get(Ndb* connection, std::string datasetName, int projectId) {
+    AnyMap args;
+    args[1] = datasetName;
+    args[2] = projectId;
+    DatasetVec datasets = doRead(connection, getColumn(1), args);
```

Comment:
is the index name `dataset_name` ?

Reply:
updated the code to use the name directly. I assume this creates and index with `projectId_name` as name... right?
https://github.com/logicalclocks/hopsworks-chef/blob/master/files/default/sql/ddl/3.3.0__initial_tables.sql#L154